packages/flutter/lib/src/widgets/form.dart

```diff
@@ -11,6 +11,7 @@ import 'dart:async';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
 
+import '../scheduler/binding.dart';
```

Comment:
I think the convention is to do relative imports for files within the same directory, and package imports otherwise.

A couple of options:

```dart
// package import
import 'package:flutter/scheduler.dart';


// using WidgetsBinding wouldn't require the scheduler import
WidgetsBinding.instance.addPostFrameCallback(...)
```